src/content/app/entity-viewer/gene-view/components/protein-image/ProteinImage.tsx

```diff
@@ -18,16 +18,14 @@ import React from 'react';
 import { scaleLinear } from 'd3';
 import classNames from 'classnames';
 
-import { Product } from 'src/shared/types/thoas/product';
-
 import transcriptsListStyles from 'src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss';
 import styles from './ProteinImage.scss';
 
 const TRACK_HEIGHT = 24;
 const PROTEIN_HEIGHT = 10;
 
 type ProteinImageProps = {
-  product: Pick<Product, 'length'>;
+  productLength: number;
```

Comment:
I guess if this component is called ProteinImage, then we can call the property proteinLength rather than productLength.

Reply:
Makes sense, done.